### DISCUSSION TITLE
issue in my notifications page get 404

### DISCUSSION THREAD

**Discussion by WaelMz** [community/community#51494] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

my notifications page my profile give me 404 page i don't understand why

**Comment by richardlhao** [comment]
Jeremy Holcomb, how can I contact you?

**Reply by WaelMz** [reply]
don't understand!

**Comment by jandresrodriguez** [comment]
Same is happening to me @richardlhao , do we already know the issue? I just get an invitation to collaborate on a private repo, and getting 404 when trying to accept the invitation too. Weird

**Reply by salluzziluca** [reply]
same here

**Reply by bandogora** [reply]
This is happening to me as well after getting an invitation.

**Reply by bandogora** [reply]
The issue is fixed for me today, not sure what changed.

**Reply by salluzziluca** [reply]
Same here

**Comment by WaelMz** [comment]
issue still on, can anyone tell me how send this issue to github  team ?

**Comment by opegamanalitarok** [comment]
If you are experiencing a 404 error on your GitHub notifications page or profile page, there could be a few reasons for this issue. Here are some potential solutions to try:

1. Check your internet connection: Ensure that you have a stable and working internet connection. If your connection is weak, your browser may not be able to load the page correctly.

2. Clear your browser cache: Clearing your browser cache can help resolve issues related to outdated or corrupted data stored in your browser. Try clearing your browser cache and cookies and then try accessing your notifications or profile page again.

3. Ensure that you are logged in: Make sure that you are logged in to your GitHub account. If you are not logged in, you will not be able to access your notifications or profile page.

4. Check your GitHub settings: Go to your GitHub settings and ensure that your notifications settings are set up correctly. Ensure that you have enabled the notifications you want to receive and that they are being sent to the correct email address.

5. Check for server issues: It is possible that there may be server issues on GitHub's end. Check GitHub's status page or social media accounts to see if there are any known issues or outages.

If none of these solutions work, you can try contacting GitHub's support team for further assistance.


Hope this helps! 🙌🙌

**Comment by WaelMz** [comment]
I am not sure how problem is resolved but it's resolved by itself after a few weeks!
